packages/cli/src/platform/apikey/create.ts

```diff
@@ -0,0 +1,38 @@
+import { arg, Command, isError } from '@prisma/internals'
+
+import {
+  getOptionalParameter,
+  getRequiredParameter,
+  platformParameters,
+  platformRequestOrThrow,
+} from '../../utils/platform'
+
+export class Create implements Command {
+  public static new(): Create {
+    return new Create()
+  }
+
+  public async parse(argv: string[]) {
+    const args = arg(argv, {
+      ...platformParameters.project,
+      '--display-name': String,
+      '-d': '--display-name',
+    })
+    if (isError(args)) return args
+    const token = getRequiredParameter(args, ['--token', '-t'], 'PRISMA_TOKEN')
```

Comment:
Aside: This env var name should probably be more PDP specific.

Reply:
I also think its suffix should match the flag name, whatever it changes to.

@luanvdw this is the token I was mentioning before.

Based on some other ongoing discussions IIUC the most accurate term here would be `PRISMA_PLATFORM_MANAGEMENT_TOKEN` (but this is not settled yet).